packages/core/src/styles/layout/layout.stories.ts

```diff
@@ -1463,6 +1505,7 @@ export const utilitiesContainers = () => {
       <cds-demo layout cds-layout="container:md">container:md (992px)</cds-demo>
       <cds-demo layout cds-layout="container:lg">container:lg (1200px)</cds-demo>
       <cds-demo layout cds-layout="container:xl">container:xl (1440px)</cds-demo>
+      <cds-demo layout cds-layout="container:fill">'container:fill' or 'fill' (width 100%)</cds-demo>
```

Comment:
are this `container:*` values new? i don't remember seeing them before...

Reply:
They have been there for a while I think. They are under the utils part of the layout docs. I think we use them in the demo layout patterns as well